Bitcoin section (#15)
mynodebtc/mynode_docs: docs/bitcoin/troubleshoot.md

# Troubleshooting Bitcoin

## Available options

> Bitcoin can run into various kinds of issues. Although MyNode aims to automatically detect these issues and fix them without user intervention, users might have to intervene. Based on the severity of the issues, there are many options available to fix common issues. Those solutions are listed below in increasing order of cost of convenience.

Reviewer: ```suggestion
Bitcoin can run into various kinds of issues. Although myNode aims to automatically detect these issues and fix them without user intervention, users might have to intervene. Based on the severity of the issues, there are many options available to fix common issues. Those solutions are listed below in increasing order of cost of convenience.
```

> - Reboot MyNode

Reviewer: ```suggestion
- Reboot myNode
```

- Reinstall Bitcoin
- Rescan blockchain
- Reindex blockchain
- Reset blockchain
- Factory reset

> ## Reboot MyNode

Reviewer: ```suggestion
## Reboot myNode
```

> MyNode can be rebooted from the settings page with the button under Device (as shown below). Rebooting mynode is the easiest way to fix common issues.

Reviewer: ```suggestion
myNode can be rebooted from the settings page with the button under Device (as shown below). Rebooting myNode is the easiest way to fix common issues.
```

![device-settings](/images/bitcoin/device-settings.png)

## Reinstall Bitcoin

> One can reinstall Bitcoin from the settings page with the dropdown tool under Advanced (as shown below). This maybe necessary when Bitcoin installation failed during an update.

Reviewer: ```suggestion
One can reinstall Bitcoin from the settings page with the dropdown tool under Advanced (as shown below). This may be necessary when a Bitcoin installation has failed during an update.
```
